index.js

```diff
@@ -32,7 +33,7 @@ CacheBreaker.prototype.cacheBreakPath = function(base, resource) {
   var extname = path.extname(resource);
   var basename = path.basename(resource, extname);
 
-  return path.join(dirname, basename + '.' + cs.substring(0, 10) + extname);
+  return url.resolve(dirname + '/', basename + '.' + cs.substring(0, 10) + extname);
```

Comment:
Interesting.  This seems like a bug in `path.resolve` to me.  Please hold -- I'm going to dive a bit deeper this evening.